kolibri/plugins/coach/assets/src/views/CoachSideNavEntry.js

```diff
@@ -0,0 +1,39 @@
+import navComponents from 'kolibri.utils.navComponents';
+import urls from 'kolibri.urls';
+import coreStrings from 'kolibri.utils.coreStrings';
+import { UserKinds } from 'kolibri.coreVue.vuex.constants';
+import baseRoutes from '../routes/baseRoutes';
+import { coachStrings } from './common/commonCoachStrings';
+
+const component = {
```

Comment:
A slight nit - maybe we call it something else now that this definitely isn't a component?

`const sideNavConfig = {`?